course/Program.cs

```diff
@@ -0,0 +1,246 @@
+                        Console.WriteLine($"Found user {firstName} {lastName} with phone {phoneNumber}");
+                        found = true;
+                    }
+                } else
+                {
+                    if (phoneNumber.Contains(searchTerm))
+                    {
+                        Console.WriteLine($"Found user {firstName} {lastName} with phone {phoneNumber}");
+                        found = true;
+                    }
+                }
+            }
+
+            if (!found)
+            {
+                Console.WriteLine("No user found!");
+            }
+
+            Wait();
+        }
+
+        static void ModifyContact()
+        {
+            Console.Clear();
+            var phoneBook = ReadPhoneBook();
+            int index = 0;
+            (string, string, string) userData = ("", "", "");
+            Console.WriteLine("Please, write full the first and the last name of the user");
+
+            Console.WriteLine("Enter Search Term...");
+            var searchTerm = Console.ReadLine();
+
+            bool found = false;
+            foreach (var (firstName, lastName, phoneNumber) in phoneBook)
+            {
+                if ($"{firstName} {lastName}".Equals(searchTerm, StringComparison.OrdinalIgnoreCase))
+                {
+                    userData = ( firstName, lastName, phoneNumber );
+                    found = true;
+                    break;
+                }
+                if (!found)
+                {
+                    index++;
+                }
+            }
+
+            if (index == phoneBook.Length && !found)
+            {
+                Console.WriteLine("No user found!");
+            } else
+            {
+                Console.WriteLine("Please, write new phone number");
+                string newPhoneNumber = Console.ReadLine();
+                userData.Item3 = newPhoneNumber;
+
+                phoneBook[index] = userData;
+
+                string[] convertedPhoneBook = new string[phoneBook.Length+1];
+                convertedPhoneBook[0] = "FirstName,LastName,PhoneNumber";
+                for (int i = 0; i < phoneBook.Length; i++)
+                {
+                    convertedPhoneBook[i + 1] = $"{phoneBook[i].Item1},{phoneBook[i].Item2},{phoneBook[i].Item3}";
+                }
+
+                try
+                {
+                    File.WriteAllLines(Filename, convertedPhoneBook);
+                }
+                catch (DirectoryNotFoundException)
+                {
+                    WriteToLocalFile(convertedPhoneBook);
+                }
+                catch (System.Security.SecurityException)
+                {
+                    WriteToLocalFile(convertedPhoneBook);
+                }
+            }
+
+            Wait();
+        }
+
+        private static (string, string, string)[] ReturnEmptyPhoneBook()
+        {
+            Console.WriteLine("There was an issue processing your file location");
```

Comment:
not critical, but your method name is **ReturnEmptyPhoneBook** but it does 2 things: writes to console and returns an empty array. It's violation of SOLID principles we're going to learn, but now it's ok